editor/editor_node.cpp

```diff
@@ -954,7 +955,8 @@ void EditorNode::_fs_changed() {
 	// FIXME: Move this to a cleaner location, it's hacky to do this in _fs_changed.
 	String export_error;
 	Error err = OK;
-	if (!export_defer.preset.is_empty() && !EditorFileSystem::get_singleton()->is_scanning()) {
+	// It's important to wait for the first scan to finish; otherwise, scripts or resources might not be imported.
+	if (!export_defer.preset.is_empty() && !EditorFileSystem::get_singleton()->is_scanning() && !EditorFileSystem::get_singleton()->doing_first_scan()) {
```

Comment:
```suggestion
	if (!export_defer.preset.is_empty() && !EditorFileSystem::get_singleton()->is_scanning()) {
```
Unnecessary after `is_scanning()` change.

Reply:
Done!